LQL/index.html

```diff
@@ -52,7 +52,9 @@
 
       <div class="container-fluid">
       <div class="row">
-      <center><p><h3>Loklak Query Language</h3></p></center>
+      <div id="header" style="margin-top: 60px; margin-bottom: 60px;">
```

Comment:
I think it's better to put this fix into it's own commit instead of combining it with the loklak url fix.